public/scss/home.scss

```diff
@@ -0,0 +1,238 @@
+/* 88888888888888888    slider   8888888888888888888888 */
+.home {
+  .slider {
+    background-color: #fff;
+    margin-bottom: 20px;
+    padding: 15px 0 6px;
+    .slider_home {
+      position: relative;
+      margin: auto;
+      .prev,
+      .next {
```

Comment:
You should avoid nesting selectors more than 3 levels.

Reply:
Done